common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -30,6 +35,10 @@ public CategoriesProvider() {
         for (Label.LabelLayer layer : Label.LabelLayer.values()) {
             PROVIDED_CATEGORIES.add(new PlaceCategory(layer));
         }
+        for (int tier = 1; tier <= 4; tier++) {
```

Comment:
Maybe `LootChestType.size()` should be used here, just in-case this is not updated, when a new loot chest tier is introduced.

Reply:
Unlikely to happen, and that will break the textures as well... But sure, it might be more clear to read.